read_input_parser_cvc.l

```diff
@@ -561,6 +628,73 @@ namespace cvc {
   } 
   if(cvc::myverbose!=0) printf("# [read_input_parser] source type set to %s\n",yytext);
 }
+<FILTERQSQ>{FLT} {
+  cvc::g_filtered_qsq = atof(yytext);
+  if(cvc::myverbose!=0) printf("# [read_input_parser] filtering momenta upto %f\n", cvc::g_filtered_qsq);
+}
+<SPINTRACE>{DIGIT}+ {
+  cvc::g_spintrace = (unsigned int)atoi(yytext);
+  if(cvc::myverbose!=0) printf("# [read_input_parser] cvc::g_spintrace set to %s\n",yytext);
+}
+<CURRENTGAMMABASIS>{FILENAME} {
+  if(strcmp(yytext,"Chiral_tmlqcd")==0){
```

Comment:
This should be parsed differently in order to be case-insensitive and consistent with the rest of the parser. Since methods using ```tolower``` and similar ones are problematic with respect to their behaviour, the best thing to do is to replicate what is done in tmLQCD:

```
^CurrentGammaBasis{EQL} BEGIN(CURRENTGAMMABASIS);

[...]

<CURRENTGAMMABASIS>ukqcd {
  [...]
}
<CURRENTGAMMABASIS>chiral_tmlqcd {
  [...]
}
```

This way, it won't matter whether this is upper, lower or mixed case.

The other thing: please define an enum with the indices for the two gamma bases and use the values here as well as elsewhere.